src/helpers/dto.ts

```diff
@@ -1,4 +1,10 @@
-export interface ICreatePerson {
-    name: string;
+export interface ICreateUser {
```

Comment:
Since this interface is used for other purposes than creating user, should't we rename it to something like `IUser`?